src/oc_erchef/habitat/hooks/run

```diff
@@ -6,10 +6,10 @@ export RELX_CONFIG_PATH="{{pkg.svc_config_path}}/sys.config"
 export VMARGS_PATH="{{pkg.svc_config_path}}/vm.args"
 export CHEF_SECRETS_DATA=$(cat {{pkg.svc_config_path}}/veil-secrets.json)
 # these are needed in order to make depselector start up
-export PATH={{pkgPathFor "core/ruby"}}/bin:{{pkgPathFor "core/bundler"}}/bin:${PATH}
+export PATH={{pkgPathFor "core/ruby26"}}/bin:${PATH}
 export LD_LIBRARY_PATH={{pkgPathFor "core/libffi"}}/lib:${LD_LIBRARY_PATH}
-export GEM_HOME={{pkgPathFor "core/bundler"}}
-export GEM_PATH={{pkgPathFor "core/bundler"}}
+export GEM_HOME={{pkgPathFor "core/ruby26"}}
```

Comment:
This really should be pointing at the bundle/vendor dir inside the erchef package?  As in, I think this line was wrong before and continues to be wrong now, but I'm trying to limit how much I change at once.